### PR TITLE
Python3 compatibility

### PR DESCRIPTION
**Pull request by Backscratcher**

### CODE REVIEW

File: media_bundler/bin_packing_test.py [hunk 1]
```diff
@@ -126,9 +126,9 @@ def testPackSequenceHeightWidth(self):
     def testRandomNoOverlap(self):
         # Not having overlap is an important invariant we need to maintain.
         # This just checks it.
-        for _ in xrange(3):
+        for _ in range(3):
```
[the remark this answers is not in the document]
Comment (samupl) [reply media_bundler/bin_packing_test.py]: Normally - we should. But in this use-case it doesn't really make that much of a difference (even up to `100` in `range()` would not benefit that much from a generator).

[the remark this answers is not in the document]
Comment (Backscratcher) [reply media_bundler/bin_packing_test.py]: I did it anyway, with the rest of the things not compatible.

File: media_bundler/jsmin.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 # */
 
 def jsmin(js):
-    from StringIO import StringIO
+    from io import StringIO
```
Comment (samupl) [inline media_bundler/jsmin.py]: I wonder why that import is inside this function...

File: media_bundler/bin_packing.py [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 import math
 
+from itertools import chain
```
Comment (ashwch) [inline media_bundler/bin_packing.py]: Let's move it to line 6, right under `import math`.

File: media_bundler/jsmin.py (hunk 2 above)
Comment (Backscratcher) [reply media_bundler/jsmin.py]: I have no idea, also there was an import that was not used. 🤷‍♂

**Pull request merged**